AnimView/frmMain.cpp

```diff
@@ -314,61 +313,61 @@ frmMain::frmMain()
   pSearchButtons->Add(new wxButton(this, ID_SEARCH_SOUND, L"Sound"), 0, wxALL,
                       1);
   pSearch->Add(pSearchButtons, 0);
-  pSearch->Add(m_lstSearchResults = new wxListBox(this, ID_SEARCH_RESULTS), 1,
-               wxEXPAND | wxALL, 1);
+  m_lstSearchResults = new wxListBox(this, ID_SEARCH_RESULTS);
+  pSearch->Add(m_lstSearchResults, 1, wxEXPAND | wxALL, 1);
 
   wxStaticBoxSizer* pFrameFlags =
       new wxStaticBoxSizer(wxHORIZONTAL, this, L"Frame Flags");
   wxBoxSizer* pFlags1 = new wxBoxSizer(wxVERTICAL);
-  pFlags1->Add(m_txtFrameFlags[0] = new wxTextCtrl(this, wxID_ANY), 0,
-               wxEXPAND | wxALL, 2);
-  pFlags1->Add(m_chkFrameFlags[0] = new wxCheckBox(this, wxID_ANY, L"2^0"), 0,
-               wxEXPAND | wxALL, 2);
-  pFlags1->Add(m_chkFrameFlags[1] = new wxCheckBox(this, wxID_ANY, L"2^1"), 0,
-               wxEXPAND | wxALL, 2);
-  pFlags1->Add(m_chkFrameFlags[2] = new wxCheckBox(this, wxID_ANY, L"2^2"), 0,
-               wxEXPAND | wxALL, 2);
-  pFlags1->Add(m_chkFrameFlags[3] = new wxCheckBox(this, wxID_ANY, L"2^3"), 0,
-               wxEXPAND | wxALL, 2);
-  pFlags1->Add(m_chkFrameFlags[4] = new wxCheckBox(this, wxID_ANY, L"2^4"), 0,
-               wxEXPAND | wxALL, 2);
-  pFlags1->Add(m_chkFrameFlags[5] = new wxCheckBox(this, wxID_ANY, L"2^5"), 0,
-               wxEXPAND | wxALL, 2);
-  pFlags1->Add(m_chkFrameFlags[6] = new wxCheckBox(this, wxID_ANY, L"2^6"), 0,
-               wxEXPAND | wxALL, 2);
-  pFlags1->Add(m_chkFrameFlags[7] = new wxCheckBox(this, wxID_ANY, L"2^7"), 0,
-               wxEXPAND | wxALL, 2);
+  m_txtFrameFlags[0] = new wxTextCtrl(this, wxID_ANY);
+  pFlags1->Add(m_txtFrameFlags[0], 0, wxEXPAND | wxALL, 2);
+  m_chkFrameFlags[0] = new wxCheckBox(this, wxID_ANY, L"2^0");
+  m_chkFrameFlags[1] = new wxCheckBox(this, wxID_ANY, L"2^1");
+  m_chkFrameFlags[2] = new wxCheckBox(this, wxID_ANY, L"2^2");
+  m_chkFrameFlags[3] = new wxCheckBox(this, wxID_ANY, L"2^3");
+  m_chkFrameFlags[4] = new wxCheckBox(this, wxID_ANY, L"2^4");
+  m_chkFrameFlags[5] = new wxCheckBox(this, wxID_ANY, L"2^5");
+  m_chkFrameFlags[6] = new wxCheckBox(this, wxID_ANY, L"2^6");
+  m_chkFrameFlags[7] = new wxCheckBox(this, wxID_ANY, L"2^7");
+  pFlags1->Add(m_chkFrameFlags[0], 0, wxEXPAND | wxALL, 2);
+  pFlags1->Add(m_chkFrameFlags[1], 0, wxEXPAND | wxALL, 2);
+  pFlags1->Add(m_chkFrameFlags[2], 0, wxEXPAND | wxALL, 2);
+  pFlags1->Add(m_chkFrameFlags[3], 0, wxEXPAND | wxALL, 2);
+  pFlags1->Add(m_chkFrameFlags[4], 0, wxEXPAND | wxALL, 2);
+  pFlags1->Add(m_chkFrameFlags[5], 0, wxEXPAND | wxALL, 2);
+  pFlags1->Add(m_chkFrameFlags[6], 0, wxEXPAND | wxALL, 2);
+  pFlags1->Add(m_chkFrameFlags[7], 0, wxEXPAND | wxALL, 2);
```

Comment:
These array assignments start to look like it could use a loop, although I can also live without it.

Reply:
Yeah, I'll live with it unrolled, they aren't that long. Two sets of 8 flags / checkboxes.